app/controllers/api/cookies_consent_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  class CookiesConsentController < BaseController
+    respond_to :json
+
+    def initialize
+      super
+      @cookie_name = 'cookies_consent'
+    end
+
+    def show
+      render json: { cookie_consent: !cookies[@cookie_name].nil? }
```

Comment:
the [key?](http://ruby-doc.org/core-2.1.5/Hash.html#method-i-key-3F) method might be a bit easier to understand than `nil?` plus a negation.

Reply:
Yes, thanks. Done. #learning_ruby

Reply:
Of course! and Ruby has plenty of methods like this one in its standard library.